### ISSUE TITLE
dynamic selection mode causing massive lag

### ISSUE THREAD

**Issue by anstosa** [VictorGrycuk/streamdeck-color-picker#13]
All window management features of Windows Explorer suffer from multi-second lag when dynamic mode is enabled

**Comment by VictorGrycuk** [comment]
Hi @anstosa, sorry for the late response.
It's been noted that the library to get the pixel colour can cause lag at higher speeds.
Sadly, I haven't had time to check on it due to lack of time.

Hopefully I will have more time in a couple of months.

**Comment by TheSimArchitect** [comment]
Yes, I tried it today, same issue. Everything lags terribly on my computer if I have the plugin assigned to a button, specially moving windows around (any application).

**Comment by VictorGrycuk** [comment]
@anstosa @TheSimArchitect 
Sorry for the delay. I finally got to sit down, and I think I found the issue.
I have uploaded a pre-release version [here](https://github.com/VictorGrycuk/streamdeck-color-picker/releases/tag/1.5.0-beta).

Let me know if the lag has been improved.

**Comment by DevAnyKR** [comment]
Please allow me to choose 10ms as 100ms, 200ms, 500ms, 1000ms. StreamDeck - .exe parameters.
and/or
plz also add a dynamic toggle function.
To operate at 10ms only when needed
